src/components/Slider/Slider.module.css

```diff
@@ -179,3 +177,14 @@
 .slider__marker:not(:first-child):not(:last-child) {
   text-align: center;
 }
+
+/**
+ * Horizontally centers the tooltip arrow respective to the tooltip.
+ */
+.slider :global(.tippy-arrow) {
+  transform: none !important;
```

Comment:
would [`unset`](https://developer.mozilla.org/en-US/docs/Web/CSS/unset) work here? or `initial`?

Reply:
Both would work, which would be `none` for transform in this case, but will change to `initial` in case a parent wants to set transform